packages/messageProxy/src/doc/doc.service.ts

```diff
@@ -37,60 +38,67 @@ export class DocService implements OnModuleDestroy {
         this.pubsub.subscribe<typeof awarenessMessageType>(awarenessMessageType, this.onChangesOrAwarenessMessage);
         this.pubsub.subscribe<typeof attachDocMessageType>(attachDocMessageType, this.onAttachDocMessage);
         this.pubsub.subscribe<typeof detachDocMessageType>(detachDocMessageType, this.onDetachDocMessage);
         this.pubsub.subscribe<typeof disconnectMessageType>(disconnectMessageType, this.onDisconnectMessage);
     }
 
-    applyChanges(connection: WebSocketConnection, payload: ChangesPayload) {
-        this.assertClientJoined(connection, payload.docId);
+    applyChanges(conn: WebSocketConnection, payload: ChangesPayload) {
+        this.assertClientJoined(conn, payload.docId);
 
         const doc = this.docs.get(payload.docId);
-        doc.broadcastDiff(connection.id, payload.changes);
+        doc.broadcastDiff(conn.id, payload.changes);
 
         // Sending changes to other instances
         this.pubsub.publish({
             type: 'changes',
             message: {
-                author: connection.id,
+                author: conn.id,
                 docId: doc.id,
                 data: payload.changes,
             },
         } as ChangesPubsubMessage);
 
-        this.actorService.sendChanges(connection.id, payload);
+        this.actorService.sendChanges(conn.id, payload);
     }
 
-    applyAwareness(connection: WebSocketConnection, payload: AwarenessPayload) {
-        this.assertClientJoined(connection, payload.docId);
+    applyAwareness(conn: WebSocketConnection, payload: AwarenessPayload) {
+        this.assertClientJoined(conn, payload.docId);
 
         const doc = this.docs.get(payload.docId);
-        doc.broadcastAwareness(connection.id, payload.awareness);
+        doc.broadcastAwareness(conn.id, payload.awareness);
 
         // Sending awareness to other instances
         this.pubsub.publish({
             type: 'awareness',
             message: {
-                author: connection.id,
+                author: conn.id,
                 docId: doc.id,
                 data: payload.awareness,
             },
         } as AwarenessPubsubMessage);
     }
 
-    async syncStart(connection: WebSocketConnection, payload: SyncStartPayload): Promise<SyncResponsePayload> {
+    async syncStart(_conn: WebSocketConnection, payload: SyncStartPayload): Promise<SyncResponsePayload> {
         return await this.actorService.syncStart(payload);
     }
 
-    syncComplete(connection: WebSocketConnection, payload: SyncCompletePayload) {
-        this.actorService.syncComplete(connection.id, payload);
+    syncComplete(conn: WebSocketConnection, payload: SyncCompletePayload) {
+        this.actorService.syncComplete(conn.id, payload);
     }
 
-    joinToDoc(connection: WebSocketConnection, docId: DocKey): JoinResponsePayload {
+    joinToDoc(conn: WebSocketConnection, docId: DocKey): JoinResponsePayload {
```

Comment:
it's nothing, but why do you prefer to rename connection -> conn ?

Reply:
`connection` is too long

I think `conn` is pretty straightforward variable name.
If you don't like it, no big deal, I can rename it